src/pykeen/datasets/cli.py

```diff
@@ -42,7 +42,7 @@ def analyze(dataset):
     try:
         import matplotlib.pyplot as plt
         import seaborn as sns
-    except ImportError as error:
+    except ImportError:
```

Comment:
This hides the original error message, doesn't it?

Reply:
the `from None` after the raise does that. Do we need the original error?

Reply:
The original error message contains the information which library was missing.

Reply:
Oh this makes sense. However, that information is shown in the stack trace, so I just had to remove `from None` in 8e3c7b7a and this will now act the way you want